script/Arcueid/Day 6/06_00_ARC06_3_2.txt

```diff
@@ -378,8 +378,8 @@ I reply to the man clad head to toe in bandages.
 [sha:f332fce4ddc3cbfbdb6f369bf930c60d2c6b3d89]{
 -- Page 302, Offset 14654.
 -- ……しかし、そうだな。過去から戻ってきたとしても、見違えるような現実など、そうはないという事か」
-Still, true enough...
- I suppose that even if you were to dredge such things up from your past, you wouldn't mistake them for reality."
+Still, I suppose so.
+ Even if you've come back from the past, there isn't much of reality you'd consider out of place."
```

Comment:
Yeah this line sucks. Assuming I'm interpreting this right, I think it might be more clearly worded along the lines of
> Even if you've come back from the past, it's not as though any part of this reality would be entirely unrecognizable.

Reply:
I think this interpretation is more safe, precisely because it is more vague.
When you're not sure, trying to overinterpret usually causes more problems than it fixes.